src/utils/date-helpers.ts

```diff
@@ -45,3 +45,15 @@ export function isBetweenToday(value: Date) {
 }
 
 export const aDayAgo = dayjs().subtract(1, 'day').toDate();
+
+export function stripTime(value: Date) {
+  return value.toISOString().substring(0, 10);
```

Comment:
Isn't it better to format without time? Sounds like this hack will break right around year 10k :P

Reply:
The problem is that since the date comes as datetime, dayjs is converting it to UTC time for some reason and for the love of me I wasn't able to get it to work using just the `.format()` method. So, stripping the time and passing just the string date formats the date correctly